src/common/CMakeLists.txt

```diff
@@ -31,6 +31,9 @@ endif()
 if(TCMALLOC_LIBRARY)
   shared_target_link_libraries(mfscommon ${TCMALLOC_LIBRARY})
 endif()
+if(JEALLOC_LIBRARY)
```

Comment:
Typo? `JEALLOC` --> `JEMALLOC`?

Reply:
Sorry, busy week - this is indeed a mistake!